docs/source/cuml-accel/limitations.rst

```diff
@@ -187,7 +187,7 @@ RandomForestClassifier
 ``RandomForestClassifier`` will fall back to CPU in the following cases:
 
 - If ``criterion`` is ``"log_loss"``.
-- If ``oob_score=True``.
+- If ``oob_score`` is a callable (custom scorer functions are not supported; only boolean values are accepted).
```

Comment:
I'd switch this to just

```suggestion
- If ``oob_score`` is a callable.
```

which better matches how we've written these notes in the rest of the doc. The extra parenthetical isn't necessary and only makes the docs more verbose IMO.

Same for below.